stock_vertical_lift_empty_tray_check/models/vertical_lift_operation_pick.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models
+from odoo.tools import float_is_zero
+
+
+class VerticalLiftOperationPick(models.Model):
+    _inherit = "vertical.lift.operation.pick"
+
+    def button_release(self):
+        """Release the operation, go to the next
+
+        By default it asks the user to inspect visually if the tray is empty.
+        """
+        icp = self.env["ir.config_parameter"]
```

Comment:
```suggestion
        icp = self.env["ir.config_parameter"].sudo()
```
otherwise is broken 99%